package.json

```diff
@@ -83,8 +83,7 @@
     "requirejs": "gulp requirejs",
     "generateDocumentation": "gulp generateDocumentation",
     "instrumentForCoverage": "gulp instrumentForCoverage",
-    "eslint": "gulp eslint",
-    "eslint-watch": "gulp eslint-watch",
+    "eslint": "eslint ./**/*.js ./**/*.html --cache",
```

Comment:
Unless I'm mistaken, `--quiet` would actually not be something we want.  We want to see any and all problems reported by `eslint`.  If quiet was the default before, then that's an oversight on my part.